ui/src/app/components/add-index-form/add-index-form.component.ts

```diff
@@ -39,6 +45,33 @@ export class AddIndexFormComponent implements OnInit {
         if (res !== '') this.selectedTableChange(res)
       },
     })
+
```

Comment:
Clicking on add index after deleting a rule is again opening delete rule side nav